tslint.json

```diff
@@ -0,0 +1,20 @@
+{
+    "defaultSeverity": "error",
+    "extends": [
+        "tslint:recommended",
+        "tslint-config-prettier",
+        "tslint-plugin-prettier"
+    ],
+    "linterOptions": {
+        "exclude": ["**/*.js"]
+    },
+    "rules": {
+        "prettier": true,
+        "interface-name": false,
+        "no-consecutive-blank-lines": [true, 2],
+        "object-literal-shorthand": true,
+        "object-literal-sort-keys": false,
+        "no-empty-interface": false,
+        "max-line-length": [true, 200]
+    }
+}
```

Comment:
Missing the new line at the end of the file.

Reply:
Done